videos/utils.py

```diff
@@ -22,7 +23,9 @@ def generate_s3_path(file_or_webcontent, website):
     elif new_filename_ext == "pdf":
         new_filename += "_transcript"
 
-    return f"/{website.s3_path.rstrip('/').lstrip('/')}/{new_filename.lstrip('/')}.{new_filename_ext}"
+    new_filename = f"{new_filename.strip('/')}.{new_filename_ext}"
```

Comment:
@fakhar-ud-din @HussainTaj-arbisoft I think you're right, but this should be `os.path.sep` to be more specific. As far as stripping, I think it's fine to strip on both sides. `s3_path` should never contain the bucket. Every `boto3` function I can think of takes the bucket and path as separate arguments, and we store our bucket names and paths separately as well. Either way, even if the path was `s3://bucket-name/etc/etc` then `strip` wouldn't remove anything from the left hand side, because it does not start with a slash:

```
>>> os.path.join("s3://bucket-name/a/test/path/".strip(os.path.sep))
's3://bucket-name/a/test/path'
```